src/chart.zoom.js

```diff
@@ -236,15 +212,33 @@ function panNumericalScale(scale, delta, panOptions) {
 	var start = scale.start,
 		end = scale.end;
 
-	if (tickOpts.reverse) {
-		tickOpts.max = scale.getValueForPixel(scale.getPixelForValue(start) - delta);
-		tickOpts.min = scale.getValueForPixel(scale.getPixelForValue(end) - delta);
+	var rangeMin, rangeMax
```

Comment:
It looks like maybe we can replace the logic from here down with a call to `rangeMinLimiter` and `rangeMaxLimiter`?